lib/contentful-management.js

```diff
@@ -28,7 +28,9 @@ export default function createClient (axios, params) {
     rateLimit: 6,
     rateLimitPeriod: 1000,
     maxRetries: 5,
-    retryOnTooManyRequests: true
+    retryOnTooManyRequests: true,
+    defaultHostname: 'api.contentful.com',
+    defaultHostnameUpload: 'upload.contentful.com'
```

Comment:
I will keep the naming since we are using `host` and `hostUpload` to have all hosts starting with the same characters. This ensures easier recognition of all the hosts in the passed configuration :)